internal/test/e2e/basic_test.go

```diff
@@ -564,6 +564,37 @@ func TestSNISync(t *testing.T) {
 	})
 }
 
+func TestVaultSync(t *testing.T) {
+	// ensure that Vaults can be synced to Kong gateway
+	cleanup := run.Koko(t)
+
+	defer cleanup()
+
+	vault := &v1.Vault{
+		Id:     uuid.NewString(),
+		Name:   "env",
+		Prefix: "test-vault-1",
+	}
+	c := httpexpect.New(t, "http://localhost:3000")
+	c.POST("/v1/vaults").WithJSON(vault).Expect().Status(http.StatusCreated)
+
+	dpCleanup := run.KongDP(kong.GetKongConfForShared())
+	defer dpCleanup()
+
+	require.Nil(t, util.WaitForKongPort(t, 8001))
```

Comment:
nit: `require.NoError` is better suited here.

Reply:
fixed